mongodbatlas/resource_mongodbatlas_project_api_key.go

```diff
@@ -24,8 +24,9 @@ func resourceMongoDBAtlasProjectAPIKey() *schema.Resource {
 		},
 		Schema: map[string]*schema.Schema{
 			"project_id": {
-				Type:     schema.TypeString,
-				Optional: true,
+				Type:       schema.TypeString,
+				Optional:   true,
+				Deprecated: fmt.Sprintf(DeprecationByVersionMessageParameter, "1.16.0"),
```

Comment:
@Zuhairahmed As this deprecation is not related to Atlas API or has any fixed date, similar to how we handled https://jira.mongodb.org/browse/INTMDB-1211 I set 3 major versions before removal. Let me know what you think.

Reply:
Can you create a jira issue for this and target 1.16.0? So we don't forget it